nimble/host/mesh/src/access.c

```diff
@@ -501,10 +501,15 @@ void bt_mesh_model_recv(struct bt_mesh_net_rx *rx, struct os_mbuf *buf)
 			if (elem->addr != rx->dst) {
 				continue;
 			}
+		} else if (BT_MESH_ADDR_IS_GROUP(rx->dst) ||
+			   BT_MESH_ADDR_IS_VIRTUAL(rx->dst)) {
+			/* find_op will find the correct model for the group */
+			goto proceed;
```

Comment:
This will work, but I'm not sure I'd have bothered with a goto/label. Instead, I'd just leave this branch empty (but with the code comment).

Reply:
I'm ok with dropping the goto. I wanted to make it clear what's the intention, but a comment is enough.